apps/web/components/NavTabs.tsx

```diff
@@ -67,7 +67,7 @@ const NavTabs: FC<NavTabProps> = ({ tabs, linkProps, ...props }) => {
                   onClick={onClick}
                   className={classNames(
                     isCurrent
-                      ? "border-neutral-900 text-neutral-900"
+                      ? "border-neutral-900 text-gray-900"
```

Comment:
It this from another PR? cc @sean-brydon

Reply:
> @CarinaWolli @emrysal is disabling instead of filtering a clearer approach? https://www.loom.com/share/a363658bff7c4e269be6a3171404a59c

@sean-brydon What do you think?

Reply:
Scratch that. It was a merge artifact. But about disabling I think is clearer and less complex to code. Instead of options disappearing.

Reply:
I agree, the options disappearing is really complex really quickly.